docs/name.com.md

```diff
@@ -0,0 +1,45 @@
+# Name.com
+
+## Configuration
+
+### Example
+
+```json
+{
+  "settings": [
+    {
+      "provider": "name.com",
+      "domain": "domain.com",
+      "host": "@",
+      "username": "username",
+      "password": "password",
+      "ttl": 300
+    }
+  ]
+}
+```
+
+### Compulsory parameters
+
+- `"domain"`
+- `"host"` is your host and can be a subdomain, `"@"` or `"*"` generally
+- `"username"`
+- `"password"`
+
+### Optional parameters
+
+- `"ttl"` is the time this record can be cached for in seconds. Name.com allows a minimum TTL of 300, or 5 minutes. Name.com defaults to 300 if not provided.
+- `"ip_version"` can be `ipv4` (A records) or `ipv6` (AAAA records), defaults to `ipv4 or ipv6`
+
+## Domain setup
```

Comment:
Although I appreciate the passion for documentation and to guide the user better 💯 I don't really see this section as necessary, I think the user is well aware he should have an account at name.com and have a domain purchased (or how to search for one).

Let's remove it unless you have a compelling reason to keep it?? 🤔